src/dstr-binding/default/async-gen-func-named-expr-dflt.template

```diff
@@ -19,7 +19,7 @@ flags: [async]
 
 var callCount = 0;
 var f;
-f = async function* g(/*{ elems }*/ = /*{ vals }*/) {
+f = async function* h(/*{ elems }*/ = /*{ vals }*/) {
```

Comment:
I've got it! There are cases defining a generator named `g`. 

e.g.: test/language/expressions/async-generator/dstr-named-ary-ptrn-elem-ary-elision-init.js

```js
var first = 0;
var second = 0;
function* g() {
  first += 1;
  yield;
  second += 1;
};


var callCount = 0;
var f;
f = async function* g([[,] = g()]) {
  assert.sameValue(first, 1);
  assert.sameValue(second, 0);
  callCount = callCount + 1;
};

f([]).next().then(() => {
    assert.sameValue(callCount, 1, 'invoked exactly once');
}).then($DONE, $DONE);
```